### ISSUE TITLE
Asgard Wrath, controllers no longer working

### ISSUE THREAD

**Issue by bossy321** [LibreVR/Revive#1484]
is it me or after the latest SteamVR update I cannot move in the game, buttons works but cannot walk or turn well

Stormland still works fine

anybody can confirm ?

thanks

**Comment by TherisValayrin** [comment]
Can confirm. Tried this morning/last night. Absolute ass for controls. I thought it was just compatibility, but glad to hear it's not always been this way

**Comment by bossy321** [comment]
if you find a way to revert to an older steam VR version, it would probably work but they dont make it easy

**Comment by AngelDark92** [comment]
That's very strange, mine are working perfectly. Try with the "early version of the next release" for revive because I think with the new changes the steamvrinput is no longer supported by 2.0.0.

**Comment by bossy321** [comment]
with the older Revive It works ok outside of the hall, in the hall its very hard to move around/impossible. I played 20+ hours prior to the new steamVR release with no issues

**Comment by randy1984** [comment]
I have the same problem. did you figure out?

**Comment by TherisValayrin** [comment]
Nope. Gonna get some new controllers soon, and see if that helps at all.

**Comment by ahmanwoods** [comment]
Index controllers are working fine for me in Asgards's w/ latest steamVR

**Comment by TherisValayrin** [comment]
I'll be getting some Index controllers within the next month, but the Vive
controllers don't work

**Comment by randy1984** [comment]
When will Vive controllers work? Next verson of Revive or SteamVR?

**Comment by TherisValayrin** [comment]
Great question! I have no clue unfortunately. Just keep at it!

**Comment by snackdadd185** [comment]
Is there a solution for this yet? Is so what it? Thanks.

**Comment by TherisValayrin** [comment]
So I ended up getting a pair of Index controllers, and those are perfect.
It gets rid of every controller related issue. The main issue now, would be
for the abysmal lag. When new enemies spawn in, it takes a hot damn second
for the Vive to catch up. Usually by that time i've died, and the game
takes so long to load already....

TLDR; Index controllers fix it

**Comment by CaptainWinks** [comment]
Can confirm as well this is happening I move slowly about 6 feet with a standard Vive controller and just stop. Sad it worked so well before.

**Comment by bossy321** [comment]
I did e-mail Steam support but they don't care and I find it really stupid that there is no option to re-install an older version of StreamVR or event stop updating. The more bitching they get maybe they will do something about it.

**Comment by blade2206** [comment]
Any update on that. Same problem for me :(

**Comment by bossy321** [comment]
i ended buying an oculus headset, works very well even on ultra quality
________________________________

**Comment by blade2206** [comment]
Quest 2 or Rift S?

For anyone who still struggling with it:
After some reasearch - you can still downgrade SteamVR to older version. 

https://www.reddit.com/r/SteamVR/comments/enhiva/is_there_any_way_to_revert_steamvr_to_a_previous/ - version from this post works great. Only downside is - you can't use Steam Console anymore to download depos. Valve managed to crush it somehow. But there is program you can use which you can find here:
https://github.com/SteamRE/DepotDownloader/releases which still can download
Last stable version works great.
After reading quick manual of this program commands that you can find in first link can be changed to:

dotnet DepotDownloader.dll -app 250820 -depot 250821 -manifest 5963171703241648110 -username "our_username" -password "your_password"
dotnet DepotDownloader.dll -app 250820 -depot 250824 -manifest 3232032662206257909-username "our_username" -password "your_password"
dotnet DepotDownloader.dll -app 250820 -depot 250827 -manifest 8495117274017070316-username "our_username" -password "your_password"

Downloaded depots can be found in app_main_folder/depots

Everything works as mentioned in the first link.

**Comment by bossy321** [comment]
Rift S
________________________________

**Comment by ahmanwoods** [comment]
Can anyone experiencing this issue try booting up SteamVR, then once it's all loaded, right click the revive tray icon, select "Quit", then restart revive?

**Comment by blade2206** [comment]
I always do that. Without it revive dosn't work at all.

**Comment by ahmanwoods** [comment]
I'm assuming you're using the "Inject" option in the tray app then?

Can you try dragging the game .exe onto C:\Program Files\Revive\ReviveInjector.exe?
